### ISSUE TITLE
security request for various remote/dbms engine query 

### ISSUE THREAD

**Issue by sophie-jeong** [ClickHouse/ClickHouse#29756]
> (you don't have to strictly follow this form)

**Use case**
 Here is remote/dbms engine query. the password value is existed in everywhere below. 
is there any plan to protect the password value in below statement ? 
-> Remote 
remote('addresses_expr', db, table[, 'user'[, 'password'], sharding_key])
remote('addresses_expr', db.table[, 'user'[, 'password'], sharding_key])
-> PostgreSQL 
CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
(  ...
) ENGINE = PostgreSQL('host:port', 'database', 'table', 'user', 'password'[, `schema`]);
-> MySQL 
CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
(    ...
) ENGINE = MySQL('host:port', 'database', 'table', 'user', 'password'[, replace_query, 'on_duplicate_clause'])

**Describe the solution you'd like**
After declaring the server variable in the config file, is it possible to designate a password for each server host and generate DDL without using a password ? and in this case, do not need to support password value in 'show create table' statement. 

**Additional context**
If anyone has a way to hide the password, please let me know. If not, do you have any development plans?

**Comment by kssenii** [comment]
Yes, it is already implemented https://github.com/ClickHouse/ClickHouse/pull/28577 but not in any release yet.
(But not for `remote` table engine though, probably can be also added)

**Comment by alexey-milovidov** [comment]
@kssenii Let's add it for `remote`.
